antsibull/cli/doc_commands/plugin.py

```diff
@@ -103,7 +103,7 @@ def generate_docs() -> int:
     error_tmpl = env.get_template('plugin-error.rst.j2')
 
     asyncio_run(write_rst(
-        '.'.join([namespace, collection]), plugin, plugin_type,
+        '.'.join([namespace, collection]), None, plugin, plugin_type,
```

Comment:
`version` needs to be set to `None`, otherwise you end up with `This plugin is part of the foo.bar collection (version Documentation Not Generated From a Collection).`